src/client/App.js

```diff
@@ -11,20 +11,33 @@ import Profile from './containers/Profile';
 import Loader from './components/Loader';
 import NotFoundPage from './containers/NotFound/notFoundPage.component';
 import Footer from './components/Footer/Footer.js';
+import { CVreviews } from './components/CVreviewsPage/CVreviews';
+import { AddResume } from './components/AddResume/AddResume';
 
 function App() {
+  const [uploadedFile, setUploadedFile] = useState('');
   const { isAuthenticated, isLoading } = useAuthentication();
   if (isLoading) return <Loader />;
   return (
     <Router>
       {location.pathname === '/sign-in' ||
       location.pathname === '/sign-up' ? null : (
-        <Header isAuthenticated={isAuthenticated} username="William Henry Gates" />
+        <Header
+          isAuthenticated={isAuthenticated}
+          username="William Henry Gates"
+        />
       )}
       <Switch>
         <Route exact path="/">
           <Home />
         </Route>
+        <AddResume
+          uploadedFile={uploadedFile}
+          setUploadedFile={setUploadedFile}
+          exact
+          path="/add-resume"
+        />
+        <CVreviews uploadedFile={uploadedFile} exact path="/cv-reviews" />
```

Comment:
i think this should be added to backend.  it shouldn't be displayed directly. there should be post method to fetch files

Reply:
I don't get you. Is it possible to make it clear why?As I remember, designer described the Add Resume should be functioned when the user click on it the next page (CV review page) get opened and user can see the attached CV appeared on the page.

Reply:
no hamed. it's in your issue to add url to backend please check your issue #49. beacuse i already developed the issue to fetch and render ad pdf.

Reply:
I have checked issue #49 several times. The code I've written working good and just need some developments. you have allready developed the issue to fecth and render as odf??why?I do not think it is the meaning of team working. It does'nt matter for me that working in this story more if it is needed or less. But first of all I want to hear from the supervisor or mentor.  @KavyaReddy732

Reply:
https://user-images.githubusercontent.com/54910388/107796212-e2d41980-6d59-11eb-8578-f77dee55b15d.mov

Reply:
because it was my issue and created by daniel and here is issue #83  and i didn't it on my own hamed and am sorry if i offend you in any way. if you understtod your issue then why it labled as backend and frontend (it should be labeled as only front end)

Reply:
and am sorry and thank you